website/content/index.mdx

```diff
@@ -0,0 +1,201 @@
+
+- Run urgent jobs in a timely fashion
+- Balance resource allocation between users
+- Configurable preemption policies
+
+### Enterprise-Grade Operations
+
+**Massive Scale Support**
+
+- Utilize multiple Kubernetes clusters simultaneously
+- Scale beyond single cluster limitations
+- Add and remove clusters without service disruption
+
+**Advanced Resource Management**
+
+- Resource and job scheduling rate limits
+- Detailed resource allocation controls
+
+**Comprehensive Monitoring**
+
+- Detailed analytics via [Prometheus](https://prometheus.io/) integration
+- Resource allocation and system behavior insights
+- Automatic failure detection and node removal
+
+**Production-Ready Features**
+
+- Secure authentication and authorization
+- High availability architecture
+- Automatic node failure handling
+
+## Use Cases and Success Stories
+
+### High-Performance Computing (HPC)
+
+- **Machine Learning Training**: Distribute large-scale ML training jobs across multiple clusters
+- **Scientific Computing**: Run complex simulations and data analysis workloads
+- **Financial Modeling**: Execute risk calculations and quantitative analysis at scale
+
+### Data Processing Pipelines
+
+- **ETL Workloads**: Process large datasets with parallel batch jobs
+- **Data Analytics**: Run distributed analytics jobs across multiple clusters
+- **Backup and Archival**: Coordinate large-scale data movement operations
+
+### CI/CD and Development
+
+- **Build Systems**: Distribute compilation and testing jobs
+- **Integration Testing**: Run comprehensive test suites across multiple environments
+- **Deployment Automation**: Coordinate complex deployment workflows
+
+### Production Deployment at G-Research
+
+G-Research, a leading quantitative research company, uses Armada in production to:
+
+- Process millions of jobs per day
+- Manage tens of thousands of nodes
+- Support diverse computational workloads
+- Maintain high availability and performance
+
+## Comparison with Other Schedulers
+
+### vs. Native Kubernetes Scheduler
+
+- **Scale**: Armada spans multiple clusters vs. single cluster limitation
+- **Throughput**: Millions of jobs vs. thousands with native scheduler
+- **Batch Features**: Purpose-built for batch vs. service-oriented design
+- **Fair Scheduling**: Advanced fair-use policies vs. basic priority classes
+
+### vs. Traditional HPC Schedulers (SLURM, PBS)
+
+- **Container Native**: Built for containerized workloads vs. traditional HPC
+- **Kubernetes Integration**: Leverages Kubernetes ecosystem vs. isolated systems
+- **Cloud Ready**: Designed for cloud and hybrid environments
+- **Modern APIs**: REST/gRPC APIs vs. command-line interfaces
+- **Rich Client Support**: Client libraries available for multiple languages (Go, Java, Scala, Python and .NET)
+
+## Getting Started
+
+> [!WARNING]
+> TODO: add links
+
+Ready to explore Armada? Here are your next steps:
+
+1. **Quick Start**: Try the local installation guide to get Armada running with Kind
```

Comment:
The way this is worded, they imply they should be linked to something but are not.